### PR TITLE
Add last edited column

### PR DESCRIPTION
**Pull request by anneschuth**
# Description

Add last edited column.
Links to #226 

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: amt/site/templates/projects/details_info.html.j2 [hunk 1]
```diff
@@ -23,20 +23,18 @@
...
+                    <div target="_self" class="rvo-tag rvo-tag--with-icon rvo-tag--active rvo-tag--success">
+                      <span class="utrecht-icon rvo-icon rvo-icon-bevestiging rvo-icon--lg" role="img"></span>
+                      {{ lifecycle.name }}
                     </div>
                 </td>
             </tr>
+            <tr class="rvo-table-row">
+                <td class="rvo-table-cell">{% trans %}Last updated{% endtrans %}</td>
+                <td class="rvo-table-cell">
+                    {{ last_edited|format_datetime("en") }}
```
Comment (robbertbos) [inline amt/site/templates/projects/details_info.html.j2:35]: @anneschuth can we make the formatting configurable based on the user's language setting?

Comment (anneschuth) [reply amt/site/templates/projects/details_info.html.j2:35]: Great question, I tried but failed initially. Let me try again.

Comment (robbertbos) [reply amt/site/templates/projects/details_info.html.j2]: @anneschuth did you try:


```suggestion
                    {{ last_edited|format_datetime(language) }}
```

Or maybe we can use the full version?

```suggestion
                    {{ last_edited|format_datetime(language, "full") }}
```

**Review by robbertbos — DISMISSED** [review]
LGTM

File: amt/site/templates/projects/details_info.html.j2 (hunk 1 above)
Comment (anneschuth) [reply amt/site/templates/projects/details_info.html.j2]: yes, and that works :)  It's already in this branch.

**Comment by anneschuth** [comment]
I'm replacing other last_edited dates still with this same date

**Review by ChristopherSpelt — APPROVED** [review]
lgmt

**Review by robbertbos — APPROVED** [review]
LGTM

**Review by robbertbos — DISMISSED** [review]
LGTM

**Review by robbertbos — APPROVED** [review]
